Убран робаст тулбокс и клещи из пермы.

DRAKONIR opened the pull request:
<!-- ЭТО ШАБЛОН ВАШЕГО PULL REQUEST. Текст между стрелками - это комментарии - они не будут видны в PR. -->

## Описание PR
<!-- Ниже опишите ваш Pull Request. Что он изменяет? На что еще это может повлиять? Постарайтесь описать все внесённые вами изменения! -->
Был убран из cargo.yml (лотерейный билет $$$) робастный тулбокс, который нарушает само понятие "НЕ МАППИТЬ".
![image](https://github.com/user-attachments/assets/2e89eb39-d1c5-4f39-8233-be31da00d837)
Requiem убрал из пермы спавн клещей.


**Медиа**
<!-- Если приемлемо, добавьте скриншоты для демонстрации вашего PR. Если ваш PR представляет собой визуальное изменение, добавьте
скриншоты, иначе он может быть закрыт. -->

**Проверки**
<!-- Выполнение всех следующих действий, если это приемлемо для вида изменений сильно ускорит разбор вашего PR -->
- [ ] PR полностью завершён и мне не нужна помощь чтобы его закончить.
- [ ] Я внимательно просмотрел все свои изменения и багов в них не нашёл.
- [ ] Я запускал локальный сервер со своими изменениями и всё протестировал.
- [ ] Я добавил скриншот/видео демонстрации PR в игре, **или** этот PR этого не требует.

**Изменения**
<!--
Здесь вы можете написать список изменений, который будет автоматически добавлен в игру, когда ваш PR будет принят.

В журнал изменений следует помещать только то, что действительно важно игрокам.

В списке изменений тип значка не является часть предложения, поэтому явно указывайте - Добавлен, Удалён, Изменён.
плохо: - add: Новый инструмент для инженеров
хорошо: - add: Добавлен новый инструмент для инженеров

Вы можете указать своё имя после символа :cl: именно оно будет отображаться в журнале изменений (иначе будет использоваться ваше имя на GitHub)
Например: :cl: Ian

-->

:cl:
- add: Добавлено веселье!
- remove: Убрано веселье!
- tweak: Изменено веселье!
- fix: Исправлено веселье!

Requiem4soul committed:
Фикс фикса луж

Старый фикс содержал лишь фичу, в которой маленькие лужи можно было смыть ногами, но не содержал самого фикса бага. Теперь в лужу добавляется 0.01 ед воды при взаимодействии с шваброй, из-за чего Solution лужи снова работает корректно и проходит все проверки при взаимодействии с шваброй

Requiem4soul committed:
Возвращение АВД

Исправляю свой косяк. По ошибке подумал что IAA - это ИИ корвакса. Сегодня я узнал что это АВД...

Requiem4soul committed:
Revert "Возвращение АВД"

This reverts commit 47968ce23e71319b42308bb933c8cf76b1e9ad3f.

Requiem4soul committed:
Merge pull request #51 from Requiem4soul/bugfixes

Фикс фикса луж

Requiem4soul committed:
Reapply "Возвращение АВД"

This reverts commit cb7693fadc3ef2df713df25dc3693cb177285821.

Requiem4soul committed:
Merge pull request #52 from Requiem4soul/bugfixes

Возвращение АВД

Requiem4soul committed:
Фикс кластера

1) Добавление консоли шаттла утилей
2) Добавление газодобытчиков на шаттл утилей
3) Новый бекап Кластера

Requiem4soul committed:
Merge pull request #53 from Requiem4soul/bugfixes

Фикс кластера

DRAKONIR committed:
Обнова карты Одиссей.

DRAKONIR committed:
Добавление легендарной кирки с описанием, спрайтом.

Requiem4soul committed:
Возвращение возрождения

Разработчики дуреют с таких переносов

Requiem4soul committed:
Merge pull request #54 from Requiem4soul/bugfixes

Возвращение возрождения

Requiem4soul committed:
Добавление руки боргам + фикс медицинского борга

1) Всем боргам добавлена рука
2) Медицинскому боргу включили все модули доступные
TODO: добавить СБ борга

Requiem4soul committed:
Merge pull request #55 from Requiem4soul/bugfixes

Добавление руки боргам и фикс медицинского борга

DRAKONIR committed:
Уменьшен урон легендарной кирки с 60 брута до 25.

DRAKONIR committed:
Легендарная кирка может выпасть в экспедиции с 5% шансом

DRAKONIR committed:
Спрайт легендарной кирки

Requiem4soul committed:
Починка карты

1) Не определялась сама станция из всех гридов
2) Станционный ИИ не был добавлен в доступные роли

NithaRe committed:
Fix Dublicate errors in Legendary pickaxe

NithaRe committed:
Delete Resources/Prototypes/_Orienta/Catalog/Fills/Crates/salvage.yml

NithaRe committed:
Delete Resources/Prototypes/_Orienta/Catalog/Fills/Crates/permaescape.yml

NithaRe committed:
Delete Resources/Prototypes/_Orienta/Procedural/salvage_loot.yml

NithaRe committed:
Update tags.yml

DRAKONIR committed:
Изменена строка отвечающая за текстуры кирки

DRAKONIR committed:
Merge remote-tracking branch 'origin/legendary-pickaxe' into legendary-pickaxe

Requiem4soul committed:
Фикс карты

1) Удаление лишних гридов
2) Уменьшено количество мусора
3) Спавн SE не работал вне самой станции
4) Добавление ламп в оружейку и комнату смотрителя

Requiem4soul committed:
Фикс карты

1) Убрано что все источники питания сами вырабатывают энергию и станция может обойтись без ДАМ/соляр и т.д. Кроме СМЭСа ИИ
2) Из ящика для побега убран случайный спавн космо клеща
3) Ящик с случайными предметами для побега был добавлен в перму
4) Добавлена точка появления наблюдателя

Requiem4soul committed:
Merge pull request #56 from NithaRe/legendary-pickaxe

Обновление Одиссеи и создание легендарной кирки.

Requiem4soul committed:
HotFix Карты Одиссея

Убраны вручную добавленные гриды:
1) Карго шаттла
2) Утильского шаттла
3) Трейд поста

Requiem4soul committed:
Merge pull request #57 from Requiem4soul/bugfixes

HotFix Карты Одиссея

Requiem4soul committed:
Изменено названия шаттла утилей

Раньше на консоли масс отображался как "corvax_mining"

Requiem4soul committed:
Merge pull request #58 from Requiem4soul/bugfixes

HotFix Изменено название шаттла утилей

Requiem4soul committed:
Revert "Изменено названия шаттла утилей"

This reverts commit d24c83e2260154585128d433802194496e050d0b.

Requiem4soul committed:
Fix станции Одиссея

Теперь название станции отображается

TODO: Сделать нормальное отображение в консоли сканере масс. Я всё что можно уже потыкал, 0 идей как фиксить

Requiem4soul committed:
Merge pull request #59 from Requiem4soul/bugfixes

Fix названия станции и ревёрт названия шаттла утилей

DRAKONIR committed:
Был убран робастный тулбокс с лотереи

DRAKONIR committed:
Merge remote-tracking branch 'origin/legendary-pickaxe' into legendary-pickaxe

Requiem4soul committed:
Проверка изменения маппула

Requiem4soul committed:
База + лог с проверкой

Requiem4soul committed:
Изменение маппула + проверки

Requiem4soul committed:
Изменение minPlayers

Requiem4soul committed:
Убраны ненужные проверки

Requiem4soul committed:
Изменено время для голосования

Нужно чтобы время окончания голосовало происходило в лобби и до 15 секунды до начала раунда

Requiem4soul committed:
Дефолтная карта - Одиссея

Requiem4soul committed:
Это трогать нельзя

Нужно переносить запуск голосования в старт сервера

Requiem4soul committed:
Попытка сделать голосовании в лобби

Requiem4soul committed:
Это нельзя трогать...

Requiem4soul committed:
Проверка

Requiem4soul committed:
Лишнее

Requiem4soul committed:
Финальные изменения

Requiem4soul committed:
Номер ПРа исправлен на верный

Requiem4soul commented:
У нас актуальная сейчас ветка bugfixes, а у тебя изменения к ветке "master"
![изображение](https://github.com/user-attachments/assets/d54cd347-e427-4980-92d5-169f7e5002b5)

Из-за этого у тебя итоговое изменение брокнуто и показывает что ты как будто space station 15 накодил (или удалили)
![изображение](https://github.com/user-attachments/assets/ca35f36c-93d3-4407-be25-24db937732d0)

Как сейчас быть?
1) Закрой данный ПР, но сохрани именно свои изменения, чтобы применить их в новом ПРе
2) В гите переключись на ветку "bugfixes"
3) Обнови её через Fetch origin
![изображение](https://github.com/user-attachments/assets/bb4da71f-afc1-44dc-a0e4-935d5e922bfb)
4) Перенеси туда свои изменения
5) Создай ПР, в котором в ветку "NithaRe:bugfixes" будут вносится изменения из твоей ветки "твой_форк:bugfixes"

Requiem4soul committed:
Merge pull request #61 from Requiem4soul/bugfixes

Голосование в конце раунда + маппул Ориенты

Requiem4soul committed:
Попытка понять как достать ckey или uuid

Requiem4soul committed:
В итоге метод прокладка

Из-за того, что покупка может вызывать асинхронный метод, пришлось создавать метод прокладку, который позволит вызывать асинхронно оба метода (оригинальному всё равно на это). Иного пути обхода синхронному методу вызывать асинхронный - я пока не нашёл

Requiem4soul committed:
Фикс проблемы с forcemap

Теперь forcemap до голосования отменяется, но всё ещё после и во время можно заforcemapить карту и она будет во время следующего раунда

Requiem4soul committed:
Merge pull request #62 from NithaRe/Fix-map-vote

Фикс проблемы forcemap при голосовании за карту

Requiem4soul committed:
СуперХотФикс цели с зоопарком

Я не думал что в 2 часа ночи я пойду что-то исправлять

Requiem4soul committed:
Merge pull request #63 from NithaRe/Fix-map-vote

СуперХотФикс цели с зоопарком

Requiem4soul committed:
Удалить и не вспоминать о существовании этого

Requiem4soul committed:
Revert "Удалить и не вспоминать о существовании этого"

This reverts commit 1c0d67bbb19b8e99867c95a0b3db56db7fba2a0e.

Requiem4soul committed:
Передаю Nithare

Nithare - сейчас вкладка в КПК открывает обычный аплинк. Твоя задача открыть ориенталинк наш как в старом радио ориенталинка

Requiem4soul committed:
Корректировки от ПРа для Mocviu

В ПРе для дискорд бота Mocviu были изменены механики, которые затрагивают реализаются на C#

Requiem4soul committed:
Ещё попытки, которые стоит учитывать в дальнейшем

Requiem4soul committed:
Revert "Ещё попытки, которые стоит учитывать в дальнейшем"

This reverts commit 424388e71f8ea2089d4317c82e506a95a8ccbc3a.

Requiem4soul committed:
Частичная отмена важных изменений с TODO

Отмена изменений, который много вероятно понадобятся для добавления ориенталинка в КПК

Requiem4soul committed:
Для проверки на WatchDog

Requiem4soul committed:
Добавление в loadout ориенталинка

Requiem4soul committed:
PR 63 Покупка за ориентики

TODO: Переместить Ориенталинк в КПК. Самый лучший вариант был через basepda

Requiem4soul committed:
PR 64, а не 63

Requiem4soul committed:
Merge pull request #64 from Requiem4soul/bugfixes

Покупка за ориентики

DRAKONIR committed:
Фикс добывания руд Легендарной киркой

DRAKONIR committed:
Убран дубликат прототипа

DRAKONIR committed:
Merge pull request #65 from NithaRe/legendary-pickaxe

Хотфикс легендарной кирки.

DRAKONIR committed:
Удалены спавнрейты оружия.

DRAKONIR committed:
Merge pull request #66 from NithaRe/legendary-pickaxe

Удалены спавнрейты оружия.

DRAKONIR committed:
Обновление карты Cluster.

DRAKONIR committed:
Обновление карты Одиссея. Фиксы.

DRAKONIR committed:
Возможная причина поломки пра.

DRAKONIR committed:
Возможная причина поломки пра Odyssey

DRAKONIR committed:
Убран сломанный прототип карты

DRAKONIR committed:
Merge branch 'bugfixes' of https://github.com/DRAKONIR/mapping

# Conflicts:
#	Resources/Maps/Odyssey_Orienta.yml
#	Resources/Prototypes/Maps/Odyssey_Orienta.yml

DRAKONIR committed:
Фикс карты Odyssey

# Conflicts:
#	Resources/Maps/Odyssey_Orienta.yml

DRAKONIR committed:
Merge pull request #70 from DRAKONIR/bugfixes

Фикс карты и всё с ней связанное.

Requiem4soul commented:
У нас на текущий момент активная ветка bugfixes
![изображение](https://github.com/user-attachments/assets/24f078b2-7036-4d84-b9f9-999924c4b62e)

DRAKONIR committed:
Чай, каждому на стол! Добавлены 5 сортов чая и их спрайты.
